src/screens/TaskDetailScreen.tsx

```diff
@@ -313,50 +383,60 @@ const styles = StyleSheet.create({
     alignSelf: 'flex-start',
   },
   pendingBadge: {
-    backgroundColor: '#fff8e1',
-    borderColor: '#ffb300',
+    backgroundColor: COLORS.status.pending.background,
+    borderColor: COLORS.status.pending.border,
   },
   completedBadge: {
-    backgroundColor: '#e8f5e9',
-    borderColor: '#4caf50',
+    backgroundColor: COLORS.status.completed.background,
+    borderColor: COLORS.status.completed.border,
   },
   statusText: {
     fontWeight: '600',
     fontSize: 16,
   },
   pendingText: {
-    color: '#f57c00',
+    color: COLORS.status.pending.text,
   },
   completedText: {
-    color: '#4caf50',
+    color: COLORS.status.completed.text,
   },
   buttonsContainer: {
     margin: 16,
-    marginTop: 0,
+    marginTop: 8,
   },
   button: {
     padding: 15,
-    borderRadius: 8,
+    borderRadius: 10,
     alignItems: 'center',
     marginBottom: 12,
+    flexDirection: 'row',
+    justifyContent: 'center',
+    shadowColor: '#000',
+    shadowOffset: { width: 0, height: 1 },
+    shadowOpacity: 0.2,
+    shadowRadius: 1.5,
+    elevation: 1,
+  },
+  buttonIcon: {
+    marginRight: 8,
   },
   completeButton: {
-    backgroundColor: '#4caf50',
+    backgroundColor: COLORS.button.complete,
   },
   pendingButton: {
-    backgroundColor: '#ff9800',
+    backgroundColor: COLORS.button.pending,
   },
   shareButton: {
-    backgroundColor: '#9c27b0',
+    backgroundColor: COLORS.button.share,
   },
   editButton: {
-    backgroundColor: '#2196f3',
+    backgroundColor: COLORS.button.edit,
   },
   deleteButton: {
-    backgroundColor: '#f44336',
+    backgroundColor: COLORS.button.delete,
   },
   buttonText: {
-    color: '#fff',
+    color: COLORS.button.text,
     fontSize: 16,
     fontWeight: '600',
   },
```

Comment:
Right now your margins and paddings are hard‑coded (16, 12, etc.). A spacing scale (e.g. const SPACING = { xs: 4, sm: 8, md: 16, lg: 24 }) can improve consistency across the app.